docs/05_features/10_resource_payer.md

```diff
@@ -38,7 +38,7 @@ Find below one example that illustrate how to send transaction to the blockchain
 
 ### EOSJS
 
-To send a transaction to the blockchain using javascript via the `eosjs` library, follow the [How To Send A Transaction](https://developers.eos.io/manuals/eos/latest/cleos/how-to-guides/how-to-submit-a-transaction) steps and note in the json example the resource payer information set in the ``transaction_extensions`` field.
+To send a transaction to the blockchain using javascript via the `eosjs` library, follow the [How To Set A Payer](https://developers.eos.io/manuals/eosjs/latest/how-to-guides/how-to-set-a-payer) steps and note in the json example the resource payer information set in the ``resource_payer`` object.
```

Comment:
I got paused at the "note in the json example" - for a moment I thought that there is a "note" in the JSON. But reading it twice I understood that attention is being pointed out at the properties of the `resource_payer`` object.

Will this revision make it clearer in one stroke? 

"To send a transaction to the blockchain using JavaScript via the `eosjs` library, follow the How To Set A Payer steps. In the given JSON example, the resource payer properties are set in the `resource_payer` object."

Ignore if it just my eyes seeing it differently in first go.

Reply:
follow [...] and note
two verbs

Reply:
I re-word it.